tox.ini

```
@@ -96,6 +96,14 @@ passenv = {[testenv]passenv}
 commands =
     {env:SNOWFLAKE_PYTEST_CMD} -m "not skipolddriver" -vvv {posargs:} test
 
+[testenv:noarrowextension]
+basepython = python3.8
```

Without this `tox` will install the library anyways
```suggestion
basepython = python3.8
skip_install = True
```